### PR TITLE
some visual adjustments to the fusion components

### PR DESCRIPTION
**Pull request by ingridklepsvik**

### CODE REVIEW

File: src/components/general/OverflowMenu/OverflowMenu.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
```
Comment (MartinForre) [inline src/components/general/OverflowMenu/OverflowMenu.stories.tsx]: This entire file (along with index.tsx) can be deleted

File: src/components/general/IconButton/styles.less [hunk 2]
```diff
@@ -74,6 +77,29 @@
...
+
+        &.isToggler{
+             .iconContainer{
+                 border: none;
+                 background-color: var(--color-primary-hover-alt1);
+            }
+
+              &.isActive{ 
+                    .iconContainer{
+                        background-color: var(--color-primary);
```
Comment (MartinForre) [inline src/components/general/IconButton/styles.less]: Is there a missing `{` here? 🤔

Comment (ingridklepsvik) [reply src/components/general/IconButton/styles.less]: Didn´t fint the missing {, but I fixed the indenting so its better to read.

File: src/components/general/OverflowMenu/OverflowMenu.stories.tsx (hunk 1 above)
Comment (ingridklepsvik) [reply src/components/general/OverflowMenu/OverflowMenu.stories.tsx]: Romoved Overflow stories and index.tsx.

File: src/components/feedback/Skeleton/styles.less [hunk 3]
```diff
@@ -1,12 +1,19 @@
...
+.skeleton {
     display: inline-block;
-    height: calc(var(--grid-unit) * 2px);
     background-image: linear-gradient(90deg, var(--color-black-alt4), var(--color-black-alt5), var(--color-black-alt4));
     background-size: 200% calc(var(--grid-unit) * 2px);
     animation: skeleton-slide 2s infinite forwards linear;
     border-radius: 4px;
 }
 
+.bar{
```
Comment (eskil0312) [inline src/components/feedback/Skeleton/styles.less]: Maybe add some indentation here?

Comment (ingridklepsvik) [reply src/components/feedback/Skeleton/styles.less]: Fixed indentation there

**Pull request merged**